src/commands/list.rs

```diff
@@ -46,8 +48,9 @@ pub fn list(matches: &ArgMatches<'_>, api: &mut API) -> Result<(), Box<dyn std::
       None => return Err(Box::new(APIError::BadFormat)),
     };
     let subject_line = format!("{} ({})", display_name, name);
-    println!("{}", &subject_line);
-    println!("{}", "=".repeat(subject_line.len()));
+    term.fg(term::color::CYAN).unwrap();
+    writeln!(term, "{}", subject_line).unwrap();
+    writeln!(term, "{}", "=".repeat(subject_line.len())).unwrap();
```

Comment:
Ye gods that's a lot of unwrap. Is there seriously no cleaner way to do this in Rust?

Reply:
Each are their own command and have to be unwrapped

Reply:
Ok nvm on this its fixed